txnsync/bitmask.go

```diff
@@ -73,6 +72,9 @@ func (b *bitmask) trimBitmask(entries int) {
 	switch bitmaskType {
 	case 1:
 		(*b)[0] = 1
+		for i := entries; i < (len(*b) - 1) * 8; i++ {
+			b.setBit(i)
```

Comment:
Why is this?

Reply:
Say the bitmask is for 7 entries, and we want to convey that all 7 entries are present. Then with the old logic, the bitmask would be for case 1 and be 0000001. What we want instead is 00000000, and for the byte to be omitted from being sent since it is 0.

Reply:
However, I think that this can be removed since we trim the bitmasks at the end by just using the `best size` anyway.

Reply:
Yes, I think this is not needed.

Reply:
I think this will even be wrong. 
If the original bits is 10000000, it should be sent as 01111111. With this change, it will be sent as 00000000.
This will violate the statements in func iterate:
```
			if 255>>maxJ != v {
				// The remaining of the bits must be 1
				return errIndexNotFound
			}
```